next.config.ts

```diff
@@ -2,6 +2,9 @@ import type { NextConfig } from "next";
 
 const nextConfig: NextConfig = {
   /* config options here */
+  eslint: {
+    ignoreDuringBuilds: true,
+  },
```

Comment:
I don't think lint errors will be an issue in the prod build as long as `lint` script is not executed. But to be safe, I added this config similar to [OutcomeTracker config](https://github.com/BuildCanada/OutcomeTracker/blob/e1fdff5fbf4fca6a346cc8d6293d019c32abf63b/next.config.mjs#L4-L9)

Reply:
I'd rather this be enabled if we pass